pkgs/by-name/ar/argo-expr/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "argo-expr";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "blacha";
+    repo = "argo-expr";
+    rev = "v${version}";
+    sha256 = "sha256-XQnPFzT3PRmKeAQXLzBE5R2VvXotzxmsq+u9u5iE1QA=";
+  };
+
+  vendorHash = "sha256-HGmJVxmAj9ijsWX+qJ7J9l3uO7WvXtRU2gvx2G7N7/M=";
+
```

```suggestion

  ldflags = [ "-X main.Version=v${version}" ];

```

See https://github.com/blacha/argo-expr/blob/c7266f051ba28576de4db10555b0aad45b95003b/main.go#L23

---

Also would be good to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook if possible.